Dockerfile

```diff
@@ -63,8 +62,6 @@ COPY plugins.conf /tmp/
 RUN sed -i "s/\${BLUEOCEAN_VERSION}/${BLUEOCEAN_VERSION}/g" /tmp/plugins.conf
 RUN /usr/local/bin/install-plugins.sh < /tmp/plugins.conf
 
-# add mesos plugin
-ADD https://infinity-artifacts.s3.amazonaws.com/mesos-jenkins/mesos.hpi-${MESOS_PLUG_HASH} "${JENKINS_STAGING}/plugins/mesos.hpi"
```

Comment:
After this change, how do we test a change that has not been upstreamed ?

Reply:
For dev purposes, we can still directly copy a `mesos.hpi` built on a local dev machine.

Reply:
(discussed irl) perhaps add a comment explaining on how to use a custom hpi file.